tests/php/src/DevTools/CallbackReflectionTest.php

```diff
@@ -39,8 +39,7 @@ public function setUp() {
 
 		$this->register_core_themes();
 
-		$plugin_registry = new PluginRegistry();
-		$file_reflection = new FileReflection( $plugin_registry );
+		$file_reflection = new FileReflection( new PluginRegistry(), new ReaderThemeLoader() );
 		$file_reflection->register();
 		$this->callback_reflection = new CallbackReflection( $file_reflection );
```

Comment:
I tried to replace all this with just:

```php
$this->callback_reflection = Services::get( 'injector' )->make( CallbackReflection::class );
```

But it caused other test failures.